docs/docs/configuration/overview.md

```diff
@@ -40,6 +40,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--cookie-secure` | bool | set [secure (HTTPS only) cookie flag](https://owasp.org/www-community/controls/SecureFlag) | true |
 | `--cookie-samesite` | string | set SameSite cookie attribute (`"lax"`, `"strict"`, `"none"`, or `""`). | `""` |
 | `--custom-templates-dir` | string | path to custom html templates | |
+| `--custom-sign-in-logo` | string | path to an custom image for the sign_in page logo. Use \"-\" to disable default logo. |
```

Comment:
Did we only want to support local paths for the logo image? Or also allow a direct link to where it is already hosted (e.g. CDN)?

Reply:
That's a good idea, I hadn't really thought about it. My suggestion would be to wait to see if someone wants this, or if one of us decides it's interesting enough to look into at a future date. I suspect once we get a new release and people see there's now the logo option, we they will let us know if they'd prefer HTTP links to the image

Reply:
I think also allowing a direct link (CDN) would be a smart idea...